README.md

```diff
@@ -142,6 +142,15 @@ The last configuration change we need to make is to set up a debug configuration
 
 Now that you have configured the extension, follow these steps to use hot reload and start debugging your code.
 
+### Install dependencies and build the extension
+
+Before we deploy the extension, we need to install its dependencies and build compile the code using the following commands:
+
+```shell
+npm install
+npm run compile:dev
```

Comment:
For the readme I would rather write out the actual webpack command rather than use the npm script (so new users know what it's doing at a glance).

```suggestion
webpack --mode development
```